src/store/actions/auth.js

```diff
@@ -37,9 +37,9 @@ export const logout = () => ({
   type: LOGOUT,
 });
 
-export const loginSuccess = (id, hotel_id, email, token, user_type, name) => {
+export const loginSuccess = (id, hotel_id, email, token, user_type, name, motto) => {
   const currentUser = {
-    id, hotel_id, email, token, user_type, name,
+    id, hotel_id, email, token, user_type, name, motto
   };
```

Comment:
The whole `currentUser` could have been the argument for the `loginSuccess` function :)